### DISCUSSION TITLE
Exchanging bear token for access token fails with "please update"

### DISCUSSION THREAD

**Discussion by tekkub** [timdorr/tesla-api#566] [Q&A] [answered]
Tonight I went to refresh tokens in my app and received the following response when trying to obtain an access token:

```
{"response":"unsupported_authentication_method_-_please_update_to_the_latest_version"}
```

This auth handler has worked fine up until tonight, is anyone else seeing this response?  I'm using `client_id = "81527cff06843c8634fdc09e8ac0abefb46ac849f38fe1e431c2ef2106796384"` and `client_secret = "c7257eb71a564034f9419ee651c7d0e5f7aa6bfbd18bafb5c5c033b093bb2fa3"`

**Comment by IMgoRt** [comment, marked as answer]
No longer required
https://github.com/timdorr/tesla-api/discussions/563#discussioncomment-2541773

**Comment by tekkub** [comment]
Ah that did it!  Now I guess I have to go implement token refresh...
